### PR TITLE
feat(docs): add basic documentation

### PR DESCRIPTION
**Pull request by tomsonpl**
screen shot 2017-03-31 at 09 48 59

### CODE REVIEW

File: docs/terminal.css [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+}
+
+/*----------------
+ 	Hero Section
+----------------*/
+
+.hero {
+    position:relative;
+	display: flex;
+    flex-direction: column;
```
Comment (Idered) [inline docs/terminal.css]: Please fix spacing.

File: docs/terminal.css [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+}
+
+html{
+	font: normal 16px sans-serif;
+	color: #555;
+    background: #fff !important;
+}
+
+body {
+    background: #fff !important;
```
Comment (Idered) [inline docs/terminal.css]: No important.

File: docs/terminal.css [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+	justify-content: center;
+	position: relative;
+	min-height: 40vh;
+	padding-bottom: 0px!important;
+
+}
+
+/* FINDER WINDOW */
+
+#window {
```
Comment (Idered) [inline docs/terminal.css]: Please use classes instead of id's

File: docs/terminal.css [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+    overflow: hidden;
+}
+
+/*--RED--*/
+.red {
+   background: #FF5F56;
+}
+
+/*--YELLOW--*/
+.yellow {
```
Comment (Idered) [inline docs/terminal.css]: Why this class has margin?

File: docs/terminal.css [hunk 5]
```diff
@@ -0,0 +1,233 @@
+/*-------------
```
Comment (Idered) [inline docs/terminal.css]: Please read http://cssguidelin.es/

**Comment by adamwardecki** [comment]
We've decided to keep everything in the main docs so I'm closing this PR. Will use @Idered  comments in the cli repo

**Pull request closed without merging**